src/components/Shared/AmountInput/AmountInput.tsx

```diff
@@ -14,33 +13,75 @@ export type Props = {
 
 const AmountInput: FC<Props> = ({ amount, register, errorMessage }) => {
   const { t } = useTranslation();
+  const [amountInput, setAmountInput] = useState<string>(
+    amount ? "" + amount : ""
```

Comment:
Hm, looks weird. Is the rsult just`"${amount}"`?

Reply:
Would lead to "undefined" in the input if you would use it that way and not specify amount.

Reply:
My bad, works then I leave the check ;)